scenarioo-client/app/scenario/scenario.controller.ts

```diff
@@ -275,14 +277,17 @@ function ScenarioController($filter, $routeParams,
     }
 
     function loadRelatedIssues() {
-        RelatedIssueResource.query({
-            branchName: SelectedBranchAndBuildService.selected().branch,
-            buildName: SelectedBranchAndBuildService.selected().build,
-            useCaseName: $routeParams.useCaseName,
-            scenarioName: $routeParams.scenarioName,
-        }, (result) => {
-            vm.relatedIssues = result;
-            vm.hasAnyRelatedIssues = vm.relatedIssues.length > 0;
+        RelatedIssueResource.getForStepsOverview({
+                branchName: SelectedBranchAndBuildService.selected().branch,
+                buildName: SelectedBranchAndBuildService.selected().build,
+            },
+            $routeParams.useCaseName,
+            $routeParams.scenarioName,
+        ).subscribe((relatedIssueSummary: RelatedIssueSummary[]) => {
+                vm.relatedIssues = relatedIssueSummary;
+                vm.hasAnyRelatedIssues = vm.relatedIssues.length > 0;
```

Comment:
This causes javascript errors when no related issues were found, because the service returns null and not an empty array -->
`vm.hasAnyRelatedIssues = relatedIssueSummary != null && relatedIssueSummary.length > 0;`